liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -71,6 +71,8 @@
     private static final Pattern pattern = Pattern.compile(LIBERTY_CONFIG_MAVEN_PROPS); 
 
     protected final String PLUGIN_VARIABLE_CONFIG_XML = "configDropins/overrides/liberty-plugin-variable-config.xml";
+    protected final String DEVMODE_CONFIG_XML = "configDropins/overrides/dev-mode-config.xml";
+    private final String PROJECT_ROOT_NAME = "io.openliberty.tools.projectRoot";
```

Comment:
This constant is now defined in two different files. You should find one place to define it and then use it from everywhere that it is needed.

Reply:
DeployMojoSupport extends PluginConfigSupport  extends StartDebugMojoSupport

Reply:
Not sure of your point. The same constant should not be defined in multiple places. Put it in a util class if you have to and reference it.

Reply:
You still have it defined in DeployMojoSupport and StartDebugMojoSupport.

Reply:
Based on the comments above, I think we should define it in StartDebugMojoSupport (e.g. as `protected`) and remove it from DeployMojoSupport.

Reply:
Yes, that was a note to Trevor where he could put it.